tests/cross_functional/ui/test_validation_ui.py

```
@@ -141,3 +141,57 @@ def test_ocs_operator_is_not_present(self, setup_ui_class_factory):
+        logger.info("Verify empty state message 'No external systems connected'")
+        assert empty_state["heading_text"] == "No external systems connected", (
+            f"Expected heading 'No external systems connected', "
+            f"got '{empty_state['heading_text']}'"
+        )
+        logger.info("Verify description text")
+        assert empty_state["description_text"] is not None and (
+            "Start configuring your storage platform" in empty_state["description_text"]
+        ), (
+            f"Expected description containing 'Start configuring your storage platform', "
+            f"got '{empty_state['description_text']}'"
+        )
```

Ruff (0.15.17)

[warning] Assertion should be broken down into multiple parts

(PT018)